README.md

```diff
@@ -40,11 +40,9 @@ COBOL Language Support provides the following COBOL syntax awareness features:
 ### Autocomplete
 Autocomplete speeds up the coding process by intuitively suggesting the most likely variables or paragraphs to follow existing code. The extension provides live suggestions while you type for:
 
-- COBOL keywords
-- COBOL variables
-- COBOL paragraphs
-- CICS keywords
-- DB2 SQL keywords
+- COBOL keywords, variables and paragraphs
+- CA IDMS DML statements
+- CICS and DB2 SQL keywords
```

Comment:
I think here we may mention the variables too.

Reply:
I guess we need to add 'statements" to both CICS and SQL. It supports writing those statements at the end. 
e.g.  
- CA IDMS DML keywords, statements
- CICS keywords, statements
- DB2 SQL keywords, DML and DDL statements

Reply:
It is a list of autocomplete elements only. Do we provide also the snippets for SQL and CICS? IDMS definitely no, it is still in discussion.

Reply:
Yes for both we provide snippets, dare to mention for sure